package.json

```diff
@@ -114,7 +115,7 @@
 		"@wordpress/edit-post": "^6.5.0",
 		"@wordpress/editor": "^12.7.0",
 		"@wordpress/element": "^4.6.0",
-		"@wordpress/env": "^5.8.0",
+		"@wordpress/env": "^7.0.0",
```

Comment:
In the GHA context, we install wp-env globally at the latest version. Lately, we have seen issues with new container usage in CI instead of local. This change aligns local with remote GHA containers again.